Library/Formula/stack.rb

```diff
@@ -0,0 +1,10 @@
+class Stack < Formula
+  desc "The Haskell Tool Stack"
+  homepage "https://www.stackage.org/"
+  url "https://github.com/commercialhaskell/stack/releases/download/v0.1.2.0/stack-0.1.2.0-x86_64-osx.gz"
```

Comment:
It requires older version of GHC than what Homebrew provides. I don't know how to use older one (is it possible?).

https://github.com/commercialhaskell/stack/wiki/Downloads#os-x
> Note: due to GHC bug 10322, building stack from source fails with GHC 7.10.1. This bug will be fixed in 7.10.2, but in the meantime, we recommend using GHC 7.8.4 on OS X.